src/pages/Token/index.tsx

```diff
@@ -115,6 +115,14 @@ const TokenPage = () => {
   // if we opened child token from bundle which is on sale
   const isIgnoreOffer = useMemo(() => (!(!token || !offer || (token.tokenId === offer.tokenId && token.collectionId === offer.collectionId))), [token, offer]);
 
+  const isOwner = useMemo(() => {
+    if (!selectedAccount || isFetchingOffer || isFetchingToken) return false;
+    if (offer) {
+      return isTokenOwner(selectedAccount.address, offer.seller);
+    }
+    return token?.owner ? isTokenOwner(selectedAccount.address, token.owner) : false;
```

Comment:
u can take this check inside isTokenOwner

when null/undefined passed - returns false. Will help avoid extra ternary operators